core/src/key/mod.rs

```diff
@@ -3,9 +3,7 @@
 /// crate::key::version                  !v
 ///
 /// crate::key::root::all                /
-/// crate::key::root::access::all        /*{ac}
-/// crate::key::root::access::ac         /!ac{ac}
-/// crate::key::root::access::gr         /*{ac}!gr{gr}
+/// crate::key::root::ac                 /!{ac}
```

Comment:
Should this be `/!ac{ac}`?

Reply:
Oh, absolutely! Nice catch :+1:

Reply:
Code for it seems to be correct so just the commend here needs to be corrected 👍

Reply:
Fixed it in [fa20388](https://github.com/surrealdb/surrealdb/pull/4589/commits/fa20388c600807c472e808cf8e979fe8618efe62). It was just the comment that was wrong.